packages/cli/src/commands/doctor/doctor.ts

```diff
@@ -89,6 +90,27 @@ type FlagsT = {
   contributor: boolean | void;
 };
 
+/**
+ * Given a `healthcheck` and a `platform`, returns the specific fix for
+ * it or the fallback one if there is not one (`runAutomaticFix`).
+ */
+const getAutomaticFixForPlatform = (
+  healthcheck: HealthCheckInterface,
+  platform: NodeJS.Platform,
+) => {
+  switch (platform) {
+    case 'win32':
+      return healthcheck.win32AutomaticFix || healthcheck.runAutomaticFix;
+    case 'darwin':
+      return healthcheck.darwinAutomaticFix || healthcheck.runAutomaticFix;
+    case 'linux':
+      return healthcheck.linuxAutomaticFix || healthcheck.runAutomaticFix;
+    default:
+    case 'win32':
```

Comment:
I don't think we need this? This case is already handled

Reply:
Copy pasting 🤦‍♂️